server/views/referrals/new/courseParticipations/course.njk

```diff
@@ -24,9 +24,9 @@
     name: "otherCourseName",
     value: formValues.otherCourseName,
     type: "text",
-    classes: "govuk-!-width-one-third",
+    classes: "govuk-input--width-20",
     label: {
-      text: "Enter the programme"
+      text: "Enter the programme name"
```

Comment:
Could we make this a separate commit or at least mention it in the commit message?

Reply:
We could, it could also be two separate tickets... there's a yak that needs shaving! ;)

Reply:
Let's add this to the commit message or get rid of it and tackle it in a separate card, as you say, so future readers inspecting the git history of this codebase can see when and why it changed